src/controls/shared/WidgetWithLabelUI.tsx

```diff
@@ -23,7 +23,7 @@ export const WidgetWithLabelUI = observer(function WidgetWithLabelUI_(p: {
 
     if (WidgetDI.WidgetUI == null) return <>WidgetDI.WidgetUI is null</>
     const { WidgetLineUI, WidgetBlockUI } = WidgetDI.WidgetUI(widget) // WidgetDI.WidgetUI(widget)
-    const isCollapsible = WidgetBlockUI != null
+    const isCollapsible = WidgetBlockUI != null && widget.isCollapsible
```

Comment:
at first, I wanted to use the `widget.config.isCollapsible` but I now understand that going though the `widget.isCollapsible` directly allow to customize thing further, change default, or dynamically take into account the value 🤔  I was hopefull we could remove all those isCollapsible everywhere, but I guess we maybe don't want to. any opinion ?